README.md

```diff
@@ -3,15 +3,24 @@
 </p>
 
 # Project Protocol Web
+
 Project Protocol web client. Built with Vitejs, React, and Typescript.
 
-### Getting Started
+### Getting Started - MacOS
 
 1. Clone the project
-2. From project root, run:
+2. Install and use the correct node version with:
+
+```shell
+nvm install 16.4 && nvm use 16.4
+```
+
+3. Install [pnpm](https://pnpm.io/installation#using-homebrew) via Homebrew
```

Comment:
I think this can be shortened to just the first suggested sentence to keep it as generic as possible.

```suggestion
2. Install [pnpm](https://pnpm.io/installation) for your operating system.
```